directory/directory.go

```diff
@@ -77,13 +77,15 @@ func NewDirImageSource(dir string) types.ImageSource {
 	return &dirImageSource{dir}
 }
 
-func (s *dirImageSource) GetManifest() ([]byte, string, error) {
+func (s *dirImageSource) GetManifest() (types.ImageManifest, string, error) {
 	manifest, err := ioutil.ReadFile(manifestPath(s.dir))
 	if err != nil {
 		return nil, "", err
 	}
+	// TODO(runcom): we should hide this type from the pkg API
```

Comment:
This will be eventually just an anonymous struct with methods from the interface to be complain with the return interface arg (or at least I'd love to do so)